src/nsmgr/internal/pkg/deviceplugin/utils.go

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2020 Cisco and/or its affiliates.
```

Comment:
It seems like this whole file is basically used to compute values that should probably be done as variables in Allocate... also, probably best to use path.Join rather than mucking about with + ourselves for path separators etc.